### PR TITLE
Add bootstrap and reset scripts, and fix server side auth

### PR DESCRIPTION
**Pull request by mrchrisadams**
This PR adds:

- the missing storage library for supporting uploads
- the bootstrap commands for creating users and profiles from a data structure
- a command for resetting the users and profiles stores, so we can easily clear out a constellation

### CODE REVIEW

File: cl8-web/server/src/config.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-const admin = require('firebase-admin')
+const admin = require('firebase-admin');
```
Comment (rmparr) [inline cl8-web/server/src/config.js:1]: Looks like prettier is getting ignored. I have no preference for style (well, I do, but it's not important), just trying to keep the commit noise down.

File: cl8-web/server/src/config.js [hunk 2]
```diff
@@ -12,13 +13,28 @@ const setupFirebaseAdmin = () => {
...
+    debug('storageBucket', process.env.FIREBASE_STORAGEBUCKET);
+    debug('projectId', process.env.FIREBASE_PROJECTID);
+
+    if (
+      !process.env.FIREBASE_DATABASEURL ||
+      !process.env.FIREBASE_STORAGEBUCKET ||
+      !process.env.FIREBASE_PROJECTID
+    ) {
+      throw Error("I can't find the FIREBASE environment variables. Did load them with `source path/to/env_file.sh` ?")
+    }
```
Comment (rmparr) [inline cl8-web/server/src/config.js:27]: Nice!

File: cl8-web/server/scripts/clearUsersandDatabase.js [hunk 3]
```diff
@@ -66,26 +57,26 @@ async function main() {
...
     console.log('clearing accounts and data:')
     // - remove all the user accounts
 
     try {
-      await clearFirebaseAccounts()
-    } catch (e) {
-      console.log
+      await clearFirebaseAccounts();
+    } catch (err) {
+      console.log(err);
```
Comment (rmparr) [inline cl8-web/server/scripts/clearUsersandDatabase.js:69]: Is `debug(err)` better? Same thing on line 76.

Comment (mrchrisadams) [reply cl8-web/server/scripts/clearUsersandDatabase.js:69]: Stuff with `debug` will only appear if you set the corresponding DEBUG env var. When this is failing, I figured it's more helpful to see why this is failing.

My thinking being that if you're running this on the CLI, you are more prepared to read a stacktrace, and you might not think to have set `DEBUG="cl8.*"` first.

**Pull request merged**